src/autocomplete.js

```diff
@@ -257,6 +285,30 @@ class Autocomplete extends Component {
 
   //////////////////////////// BUTTON
 
+  buttonKeyDownHandlers = {
+    ArrowDown: this.keyDownHandlers.ArrowDown,
+
+    ArrowUp: this.keyDownHandlers.ArrowUp,
+
+    Enter(event) {
```

Comment:
Actually, I think I prefer "Enter" on the button to select the highlighted item.

I'd love to hear what @marcysutton has to say on the subject of accessibility. Right now you can hit the spacebar on the button to toggle the menu and the up/down arrow keys to change the highlighted item, then the enter key to select an item. If we add this event handler, I feel like the Enter key has too much going on:

1. It can open the menu
2. It can close the menu
3. It can select the highlighted item.

I'd rather just have the enter key select the highlighted item if one is highlighted (like it is currently). [Here's the interaction](https://codesandbox.io/s/9rWEmzEv8) as it stands today: 

![a11y](https://user-images.githubusercontent.com/1500684/28931817-985f6030-7834-11e7-9185-2889af9cfe7c.gif)